packages/jupyter-ai/src/components/chat.tsx

```diff
@@ -216,9 +218,20 @@ export function Chat(props: ChatProps): JSX.Element {
                   <Box />
                 )}
                 {view === ChatView.Chat ? (
-                  <IconButton onClick={() => setView(ChatView.Settings)}>
-                    <SettingsIcon />
-                  </IconButton>
+                  <Box sx={{ display: 'flex' }}>
+                    <Tooltip title="New Chat">
```

Comment:
The UI should use sentence case (`New chat`) instead of title case (`New Chat`).